lib/rubygems/source.rb

```diff
@@ -79,7 +79,7 @@ def dependency_resolver_set # :nodoc:
       uri
     end
 
-    bundler_api_uri = enforce_trailing_slash(fetch_uri)
+    bundler_api_uri = enforce_trailing_slash(fetch_uri) + "./versions"
```

Comment:
I'm curious about why the `/./` is put into the url. Am I misunderstanding the code?

Reply:
I used the same style that was used previously with the old API before https://github.com/rubygems/rubygems/commit/c9d157defb1783a7c5f5b890916cc3349dd1ffb7#diff-3a6811b74628a1ec350eb51728ad8c87e8110e8fb4d8938567dc50e9dcc0958eL83, but I don't think it's necessary, let me remove it.